app/data/data.js

```diff
@@ -257,3 +257,24 @@ export const TeamMembers = [
     name: "Lola Jackson",
   },
 ];
+
+export const PillLabels = [
+  {
+    name: "softStory",
+    trueData: "Non-Compliant",
+    falseData: "Compliant",
+    noData: "No Data",
+  },
+  {
+    name: "liquefaction",
+    trueData: "High Hazard",
+    falseData: "Low Hazard",
```

Comment:
perhaps "Not in Zone" or "Outside of Zone" instead of "Low Hazard"? (until API changes)

Reply:
(as you suggested)